terms.html

```diff
@@ -185,4 +185,10 @@
 does not use the term URI or IRI because those terms have been deemed to be
 confusing to Web developers.
   </dd>
+  <dt><dfn data-lt="well-formed">well-formed</dfn></dt>
+  <dd>
+A well-formed document is a conforming document, 
+and an instance of the core data model defined via a context,
+which produces no JSON-LD processing errors.
```

Comment:
We should work well-formed-ness into the definition of a conforming document, and then update all mentions of "well-formed" in the specification to "conforming document". Let's try to reduce vocabulary here, if possible.